src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+### UX/UI
+
+For Ethereum to onboard more people the UX/UI must be improved. This requires designers and UI and product experts to re-examine the design of wallets and apps.
+
+#### Background reading
+
+[Ethresear.ch UX/UI](https://ethresear.ch/c/ui-ux/24)
+
+[Ethereum.org community page](https://ethereum.org/en/learn/)
+
+#### Recent research
+
+
+## Economics
+
+There are complex crypto-economic factors relating to Ethereum's native asset (ether) and the tokens built on top of it (for example NFTs and ERC20 tokens).
+
+#### Background reading
+
+[Rollup economics from first principles](https://barnabe.substack.com/p/understanding-rollup-economics-from?utm_source=url)
+
+[Robust Incentives Group](https://ethereum.github.io/rig/)
+
+
+#### Recent research
+
+[Empirical analysis of EIP1559]([https://arxiv.org/abs/2110.04753)
+
+[MEV auctions](https://ethresear.ch/t/mev-auction-auctioning-transaction-ordering-rights-as-a-solution-to-miner-extractable-value/6788)
+
+[Circulating supply equilibrium](https://ethresear.ch/t/circulating-supply-equilibrium-for-ethereum-and-minimum-viable-issuance-during-the-proof-of-stake-era/10954#6-enforcing-minimum-viable-issuance-with-a-variable-base-reward-factor-19)
+
+[Quantifying MEV: How dark is the forest?](https://arxiv.org/abs/2101.05511)
+
+[EIP 1559: One month later](https://arxiv.org/abs/2110.04753)
+
+### PoS incentives
+
+When Ethereum transitions to proof of stake, its native asset (ether) will be used as collateral by validators. The cryptoeconomics of this determines the security of the network. Sophisticated validators may be able to exploit the nuances of the incentive layer to theior own advantage by extracting MEV or launching explicit attacks. 
+
+#### Background reading
+
+[Robist Incentives Group: PoS incentives](https://ethereum.github.io/beaconrunner/)
+
+#### Recent research
+
+[Vitalik on PBS](https://notes.ethereum.org/s3JToeApTx6CKLJt8AbhFQ#Hybrid-PBS-can-we-use-proposers-only-for-inclusion-of-last-resort)
+
+[Three Attacks on PoS Ethereum](https://arxiv.org/abs/2110.10086)
+
+[Low-cost attacks on Ethereum 2.0 by sub 1/3 stakeholders](https://arxiv.org/abs/2102.02247)
+
+
+### Liquid staking and derivatives
+
+Liquid staking allows users with less than 32 ETH to receive staking yields by swapping ether for a token representing staked ether that can be used in DeFi. However, the incentives and market dynamics associated with liquid staking as well as its effect on Ethereum's security (e.g. centralization risks).
+
+#### Background reading
+
+[Ethresear.ch liquid staking](https://ethresear.ch/search?q=liquid%20staking)
+
+
+#### Recent research
+
+[Handling withdrawals from Lido](https://ethresear.ch/t/handling-withdrawals-in-lidos-eth-liquid-staking-protocol/8873)
+
+[Withdrawal credentials](https://ethresear.ch/t/withdrawal-credential-rotation-from-bls-to-eth1/8722)
+
+### DeFi
```

Comment:
Barnabe mentioned DeFi doesn't really have to do with economics research (at least as he sees it, which is maybe more on protocol economics). While there may be economics involved with e.g. tokenomics or economic mechanisms of a DeFi protocol, I tend to agree with him - DeFi fits betters under Applications given it's on the application layer.